scripts/apps/relations/services/RelationsService.ts

```diff
@@ -1,14 +1,20 @@
 import {IArticle} from 'superdesk-interfaces/Article';
 
-RelationsService.$inject = ['archiveService'];
+RelationsService.$inject = ['archiveService', 'mediaIdGenerator'];
 
-export function RelationsService(archiveService) {
-    this.getRelatedItems = function(item: IArticle) {
+export function RelationsService(archiveService, mediaIdGenerator) {
+    this.getRelatedItems = function(item: IArticle, mediaFields: object) {
```

Comment:
Type annotations are not mandatory, and when unsure it's best to not use them at all. Adding type `object` isn't much more helpful than not adding a type annotation at all since everything in JavaScript is an object - dictionary, array, set and map are all objects. The correct annotation would be `Array<number>`